.github/workflows/ci.yml

```diff
@@ -110,10 +113,9 @@ jobs:
           mkdir build
           cd build
           cmake .. -DCMAKE_INSTALL_PREFIX=../install \
-                   -DCMAKE_C_FLAGS="-cpp --coverage -O0" \
-                   -DCMAKE_Fortran_FLAGS="-cpp --coverage -O0" \
-                   -DCMAKE_EXE_LINKER_FLAGS="--coverage" \
-                   -DCMAKE_SHARED_LINKER_FLAGS="--coverage"
+                   -DCMAKE_BUILD_TYPE=Debug \
+                   -DTESTING=ON -DJULIA_PROJECT_PATH=../libtrixi-julia \
```

Comment:
This activates building the tests. One has to specify the path to the Julia project such that Trixi.jl can run.

Reply:
Sounds reasonable to me 🤷‍♂️

Reply:
Cool!